cli/rg-client-test/main.go

```diff
@@ -12,17 +12,22 @@ import (
 )
 
 func main() {
-	cli, err := rgclient.CreateUnified()
+	cli, err := rgclient.CreateUnifiedWithOptions(&rgclient.Options{TokenFile: "/tmp/k8s_token"})
```

Comment:
I guess you want to specify a const for this and I don't see how a fixed file can help in a test as you need to write the valid token into the file somehow